bin/release

```diff
@@ -3,6 +3,8 @@
 declare(strict_types=1);
 
 use Leviy\ReleaseTool\Console\Application;
+use Leviy\ReleaseTool\Vcs\MissingConfigurationException;
```

Comment:
The configuration is not Vcs specific and should not be located in that namespace.